approved/j750/flow_control_flow.txt

```diff
@@ -114,24 +118,24 @@ DFF 1.1	Flow Table
 						Test	test4	test4						Fail									flag-true	ifallpb1_C2A232F_PASSED_AND_ifallpb2_C2A232F_PASSED				
 						logprint	Test_that_group-level_dependencies_work																					
 						Test	grp1_test1	grp1_test1			5			None		grp1_C2A232F_FAILED_0												
-						Test	grp1_test2	grp1_test2			5			None		grp1_C2A232F_FAILED_1												
-						Test	grp1_test3	grp1_test3			5			None		grp1_C2A232F_FAILED_2												
 						flag-true	grp1_C2A232F_FAILED																flag-true	grp1_C2A232F_FAILED_0				
+						Test	grp1_test2	grp1_test2			5			None		grp1_C2A232F_FAILED_1												
 						flag-true	grp1_C2A232F_FAILED																flag-true	grp1_C2A232F_FAILED_1				
+						Test	grp1_test3	grp1_test3			5			None		grp1_C2A232F_FAILED_2												

```

Comment:
Logically the same as the original, its just that setting the common flag based on the test-specific one now occurs next to each test rather than running all in a row at the end.